clients/rospy/src/rospy/topics.py

```diff
@@ -1348,7 +1348,7 @@ def get_topics(self):
         @return: list of topic names this node subscribes to/publishes
         @rtype: [str]
```

Comment:
This return type is a bit of a fib, I think, unless I don't understand the documentation syntax.